api/types/saml.go

```diff
@@ -487,6 +501,12 @@ func (o *SAMLConnectorV2) CheckAndSetDefaults() error {
 			return trace.BadParameter("need roles field in attributes_to_roles")
 		}
 	}
+
+	if o.GetPreferredRequestBinding() != "" {
+		if !slices.Contains([]string{SAMLRequestHTTPRedirectBinding, SAMLRequestHTTPPostBinding}, o.GetPreferredRequestBinding()) {
```

Comment:
can we make a array closer to https://github.com/gravitational/teleport/pull/54605/files#diff-49e649c498e77759877692d043098aa711bccd469379c3149f53b2ce916bd995R453 with valid values ?